governance/sig-governance.md

```diff
@@ -0,0 +1,81 @@
+# SIG Governance
+
+This charter details on the governance guidelines of TiDB SIGs. It will be updated as needed to meet the current needs of the TiKV project.
```

Comment:
```suggestion
This charter details on the governance guidelines of TiDB SIGs. It will be updated as needed to meet the current needs of the TiDB Cimmunity.
```